user/user.go

```diff
@@ -0,0 +1,347 @@
+}
+
+// Create creates a new TrueVault User. The username given must be unique to ACTIVATED and LOCKED Users for an
+// Account. Upon creation, both an API_KEY and an ACCESS_TOKEN will be automatically vended to the user. For security
+// reasons, the API_KEY will only be shown upon creation or via the TrueVault Management Console for the account’s
+// administrators. If group_ids is provided, the newly created user will be added to all given groups. The user making
+// the request must have the C Group::GROUPID::GroupMembership::.* or U Group::GROUPID permission for all given groups.
+// Please see authorization for more information regarding recommendations for API_KEY and ACCESS_TOKEN usage.
+//
+// @param username – string(req’d) - username for the User being created
+// @param password – string(optional) - password for the User being created. If created without a password, the user
+//					 can’t authenticate using the login endpoint, but it can still have an API key. This allows creating
+//		 			 service accounts for backups or other server-to-TrueVault communication.
+// @param attributes – b64 string(optional) - base64 encoded JSON document describing the User attributes
+// @param groupIds   – (optional) - list of group IDs where the new user will be placed
+// @param status     – (optional) - the user’s status, one of ACTIVATED (default), PENDING, or LOCKED
+// @param accessTokenNotValueAfter – (optional) - expiration time of generated access token
+func (u *User) Create(ctx context.Context, username, password, attributes string, groupIds []string, status Status, accessTokenNotValueAfter time.Time) (TvUser, error) {
+	if username == "" {
+		return TvUser{}, errors.New("username required to create user")
+	}
+
+	data := url.Values{}
+	data.Set("username", username)
+
+	if password != "" {
+		data.Set("password", password)
+	}
+
+	if attributes != "" {
+		data.Set("attributes", attributes)
+	}
+
+	if groupIds != nil {
+		data.Set("group_ids", strings.Join(groupIds, ","))
+	}
+
+	// TODO: Force empty string to be invalid. Should default (zero) to ACTIVATED
+	if status != "" {
+		data.Set("status", string(status))
+	}
+
+	if !accessTokenNotValueAfter.IsZero() {
+		data.Set("access_token_not_value_after", accessTokenNotValueAfter.String())
+	}
+
+	buf := new(bytes.Buffer)
+	if err := json.NewEncoder(buf).Encode(data); err != nil {
+		return TvUser{}, err
+	}
+
+	req, err := u.NewRequest(ctx, http.MethodPost, u.URLBuilder.CreateUserURL(), gotruevault.ContentTypeApplicationJSON, buf)
+	if err != nil {
+		return TvUser{}, err
+	}
+
+	var msg CRUDResponse
+	return msg.User, u.Do(req, &msg)
+}
+
+// List returns all Users belonging to an Account.
+//
+// status – string(optional, default: ‘ACTIVATED’) - comma separated list of statuses (inclusive). Accepts any
+//			combination of ACTIVATED, DEACTIVATED, or LOCKED.
+// full – boolean(optional, default: ‘false’) - return User attributes and Group IDs. Note: If true, then this endpoint
```

Comment:
Comment mentions about this param defaulting to `false`. But since we require to pass this param, we can remove the `default` comment